test/renate/models/test_renate_module.py

```diff
@@ -198,11 +198,11 @@ def test_renate_model_multihead_save_and_load(tmpdir, test_case, test_cls):
 def test_renate_model_add_params(test_case):
     model, train_dataset, _ = test_case
     model_updater = pytest.helpers.get_simple_updater(model)
-    model = model_updater.update(train_dataset=train_dataset, task_id="second_task")
+    model_updater.update(train_dataset=train_dataset, task_id="second_task")
     assert model._tasks_params_ids == {TASK_ID, "second_task"}
```

Comment:
We previously had `Learner.on_model_update_end` return the final model (which did nothing but return `Learner._model`). Now `on_model_update_end` is no longer called explicitly by us, but by the checkpointing callback. That's why I removed the return arg and just relied on the assumption that the model is updated in-place.

We could have `model_updater.update` return the `model`, but this would be pretty much exactly the same, since it could only return `ModelUpdater._model`.

Not sure what the best solution is here.